devops/dev_bootstrap.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -eux
+
+SCRIPT_FILE_PATH=$(readlink -f "${0}")
+SCRIPT_FOLDER=$(dirname "${SCRIPT_FILE_PATH}")
+MAIN_FOLDER="${SCRIPT_FOLDER}/.."
+LMSAPP_FOLDER="${MAIN_FOLDER}/lms"
+LMSWEB_FOLDER="${LMSAPP_FOLDER}/lmsweb"
+CONFIG_FILE_PATH="${LMSWEB_FOLDER}/config.py"
+CONFIG_EXAMPLE_FILE_PATH="${LMSWEB_FOLDER}/config.py.example"
+DB_BOOTSTRAP_FILE_PATH="${LMSAPP_FOLDER}/lmsdb/bootstrap.py"
+
+if ! (test -f "${CONFIG_FILE_PATH}"); then
+  echo "Creating config from template"
+  cp "${CONFIG_EXAMPLE_FILE_PATH}" "${CONFIG_FILE_PATH}"
+  echo "Writing secret key to config"
+  echo "SECRET_KEY = \"$(python -c 'import os;print(os.urandom(32).hex())')\"" >>"${CONFIG_FILE_PATH}"
+else
+  echo "Config already exists"
+fi
+
+echo "Installing prod requirements"
+pip3 install --user -r "${MAIN_FOLDER}/requirements.txt"
+echo "Installing dev requirements"
+pip3 install --user -r "${MAIN_FOLDER}/dev_requirements.txt"
+
+echo "Creating local SQLite DB"
+python3 "${DB_BOOTSTRAP_FILE_PATH}"
```

Comment:
Many distros have no `python3` executable. Check if `python3` exists, if not, try `python` instead  (same for `pip`)

Reply:
I don't think that our code supports python2 so it's not relevant, but as I said in the previous comment, we should install venv for this project in the dev-bootstrap

Reply:
My understanding from Yam's comment is that there are distros which python 3's executable is `python`. This is not meant to support python 2